roles/wordpress/tasks/main.yml

```diff
@@ -100,6 +100,7 @@
     args:
       chdir: '/home/{{ remote_web_user }}/www/{{ item.name }}'
     register: wp_installed
+    ignore_errors: true
```

Comment:
why this flag?

Reply:
For me the `TASK [wordpress : Is WordPress already installed]` was always failing with and error like `"non-zero return code", "rc": "1"` and breaking the play, although it still puts the right value in the `wp_installed` and it is being verified later in the next task and, really, able to / should continue. So I decided that this flag is appropriate here. Not sure why Ansible fails there if wp cli returns "1".